XenAdmin/TabPages/PhysicalStoragePage.resx

```diff
@@ -121,84 +121,102 @@
   <data name="panel1.Anchor" type="System.Windows.Forms.AnchorStyles, System.Windows.Forms">
     <value>Top, Bottom, Left, Right</value>
   </data>
-  <data name="columnHeader1.Text" xml:space="preserve">
-    <value>Name</value>
+  <metadata name="columnImage.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+    <value>True</value>
+  </metadata>
+  <data name="columnImage.HeaderText" xml:space="preserve">
+    <value />
   </data>
   <assembly alias="mscorlib" name="mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
-  <data name="columnHeader1.Width" type="System.Int32, mscorlib">
-    <value>160</value>
+  <data name="columnImage.Width" type="System.Int32, mscorlib">
+    <value>5</value>
   </data>
-  <data name="columnHeader2.Text" xml:space="preserve">
+  <metadata name="columnName.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+    <value>True</value>
+  </metadata>
+  <data name="columnName.HeaderText" xml:space="preserve">
+    <value>Name</value>
+  </data>
+  <data name="columnName.MinimumWidth" type="System.Int32, mscorlib">
+    <value>60</value>
+  </data>
+  <metadata name="columnDescription.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+    <value>True</value>
+  </metadata>
+  <data name="columnDescription.HeaderText" xml:space="preserve">
     <value>Description</value>
   </data>
-  <data name="columnHeader2.Width" type="System.Int32, mscorlib">
-    <value>180</value>
+  <data name="columnDescription.MinimumWidth" type="System.Int32, mscorlib">
+    <value>60</value>
   </data>
-  <data name="columnHeader3.Text" xml:space="preserve">
+  <metadata name="columnType.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+    <value>True</value>
+  </metadata>
+  <data name="columnType.HeaderText" xml:space="preserve">
     <value>Type</value>
   </data>
-  <data name="columnHeader3.Width" type="System.Int32, mscorlib">

```

Comment:
We should keep the Width setting for each column. Otherwise, all columns are set to same width by default (now it is 127 each, which doesn't look good.).